package.json

```diff
@@ -29,7 +29,7 @@
     "changeset": "changeset && pnpm install",
     "clean": "pnpm --recursive --parallel clean",
     "clean:nodemodules": "find . -type d -name \"node_modules\" -exec rm -fr {} +",
-    "dev": "pnpm --recursive --parallel dev",
+    "dev": "pnpm --recursive --parallel --filter @webstone/cli --filter create-webstone-app --filter @webstone/core dev",
```

Comment:
Changed the script here again, until we're having a proper solution

Reply:
Oh... I opened #307 thinking it was in `main` but realized it's from this PR :).

Let me try something to make this work.